configure

```diff
@@ -527,8 +527,11 @@ def configure_node(o):
   o['variables']['clang'] = 1 if is_clang else 0
 
   if not is_clang and cc_version != 0:
-    o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
-
+    try:
+        o['variables']['gcc_version'] = 10 * cc_version[0] + cc_version[1]
+    except IndexError:
```

Comment:
If we are expecting at least one element then we can do `if len(cc_version) == 2: ... else: ...`